easybuild/easyconfigs/g/GMP/GMP-6.0.0a-intel-2015a.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'GMP'
+version = '6.0.0a'
+
+homepage = 'http://gmplib.org/'
+description = """GMP is a free library for arbitrary precision arithmetic, 
+operating on signed integers, rational numbers, and floating point numbers. """
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'optarch': True, 'pic': True, 'precise': True}
```

Comment:
It's weird. In #1419 it didn't work it just `'precise': True`. It seems `pic` and `optarch` come to the rescue?